proto/def.bzl

```diff
@@ -69,6 +69,18 @@ def get_imports(attr, importpath):
 
 def _go_proto_aspect_impl(_target, ctx):
     attr = ctx.rule.attr
+    for attr_name in ["importpath", "_go_context_data"]:
+        if not hasattr(attr, attr_name):
+            fail("while processing go_proto_library deps: " +
```

Comment:
Please use a single triple quoted string constant with newlines escaped via backslashes for legibility.